src/neo/SmartContract/Native/OracleContract.cs

```diff
@@ -198,7 +199,7 @@ private void Request(ApplicationEngine engine, string url, string filter, string
 
             //Mint gas for the response
             engine.AddGas(gasForResponse);
-            GAS.Mint(engine, Hash, gasForResponse, false);
+            _ = GAS.Mint(engine, Hash, gasForResponse, false);
```

Comment:
this call should be awaited. Either Request should return ContractTask or we need a try/catch block